Misc/NEWS.d/next/Tests/2018-09-04-15-16-42.bpo-34579.bp4HdM.rst

```diff
@@ -0,0 +1,2 @@
+Add AIX specific DEFAULT_CONFIG
```

Comment:
I would prefer to say: Fix test_embed on AIX.

Maybe put your full name, it's up to you. But we usually write "Patch by xxx."